JourneysSquad/JourneysSquadTests/Model/BillValidationErrorTests.swift

```diff
@@ -0,0 +1,81 @@
+import XCTest
+
+final class BillValidationErrorTests: XCTestCase {
+    func testErrorDescriptionPresentsInAPI() {
+        _ = BillValidationError.tooLongPersonName(name: "someLongName").errorDescription
+        _ = BillValidationError.emptyPersonName(name: " ").errorDescription
+        _ = BillValidationError.negativeAmountOfMoney(number: -10).errorDescription
+        _ = BillValidationError.moreThanTwoDecimalPlacesForMoney(amount: 10.111).errorDescription
+        _ = BillValidationError.invalidSumOfBill(sum: 0)
+        _ = BillValidationError.incorrectEstimatedSumOfTheBill.errorDescription
+    }
+
+    func testErrorDescriptionReturnsCorrespondingMessageForTooLongPersonNameError() {
+        let name = "test"
+
+        let expectedErrorDescription = "The length of the person's name exceeds the maximum allowed length: \(name)."
+
+        let actualErrorDescription = BillValidationError.tooLongPersonName(name: name).errorDescription
+
+        XCTAssertEqual(actualErrorDescription, expectedErrorDescription)
+    }
+
+    func testErrorDescriptionReturnsCorrespondingMessageForEmptyPersonNameError() {
+        let name = " "
+
+        let expectedErrorDescription = "The name: \(name) contains only spaces or no one sign."
+
+        let actualErrorDescription = BillValidationError.emptyPersonName(name: name).errorDescription
+
+        XCTAssertEqual(actualErrorDescription, expectedErrorDescription)
+    }
+
+    func testErrorDescriptionReturnsCorrespondingMessageForNegativeAmountOfMoneyError() {
+        let amount: Decimal = -4
```

Comment:
What do you test here?
Will the test pass if you change the line: 

```suggestion
        let amount: Decimal = 1
```

?

Reply:
It was discussed on call